types/jail/pw_auth.pp

```diff
@@ -0,0 +1,3 @@
+# @summary Password Authentication setting for one SFTP jail.
+#
+type Sftp_jail::Jail::Pw_auth = Variant[Boolean, Enum['yes', 'no']]
```

Comment:
If I remember correctly, that's because it was `'yes'` and `'no'` before, so I did want to keep it compatible. I can change it to Boolean, simplifies the code. :)

Reply:
I simplified it: https://github.com/voxpupuli/puppet-sftp_jail/pull/69/commits/f640d9960b1ed0ee5de38f21a4c7cfd2f502af88